Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/icx-d-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester
ls: cannot access '/root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester': No such file or directory
[8mha:////4MftwC6NxviNjDdpfna9VBlJkoMp8EhRTGL2zcTKdEZ0AAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77aCbP8Brmfz270AAAA=[0m[Pipeline] }
[8mha:////4Kr9K7llyIIfXzIi7fTbyam3A+VyjB0I1hAyH+fEbXLCAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIMlYVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL0tBtv8Aj1Msdr0AAAA=[0m[Pipeline] // script
[8mha:////4FI2+bSiptyXTIERd0s3FW0AIUAVzDIQC15QCQE4LYPvAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRFDSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPyRmLRr0AAAA=[0m[Pipeline] }
[8mha:////4EIF7pCgSAhTUJ+YeVF144RCQBElcAy/oq5IQm3P5r/FAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRQlSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPw2Hi4r0AAAA=[0m[Pipeline] // stage
[8mha:////4O65h0jm98EO51SGQmOdf6FpfngMUfFS1Hkxj+qIATCfAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjpHSIitZywwtMYowT6y7YF5KKF/E1/oAhEhVb7ay0mucLyhThQNHJzmLvMTVeDmF0ucmJYn8JNMmOzrIhTBSs1HbS1NrjgprYwhJRwErB2mITKHl0DBvVmbupgkFXnTjmba+g8O0NHiBUNrOJ/IU5wvY6tonw5/krgHlgEDVDsas/x/INoKa7ysEAAAA=[0m[Pipeline] stage
[8mha:////4Cwh2bfnyXtsNeYvUrnMhkgZn3DOCE3ZIocZCmHhDj1kAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUSDAhpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD5WQ+PfCAAAA[0m[Pipeline] { (git clone)
[8mha:////4CFOXQFtiiA+7tA4CXv18jhcFszpIgsA6m3wBj5vXwoxAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiCmgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHN7LjSKW+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/Jiz1028/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4JxnGUheoVbdA4jH7w1zuxcsCY2poKXOhDnLMn7NKJYFAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiCkSBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGOY4V7vgAAAA==[0m[Pipeline] }
[8mha:////4D4+5H4RzROPZTW0wBHD+XjMhoO5mPw9EWUJAbCIJpuGAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiJAokRJXWcsMLTGyME+vO2A5JxYv4Gn8gEImKLVY728zzBcsU4cjRYmuoc5Qah8H3dlo4cOwungds+YwNU2JvUJpBsjb1jJKzgTlFCQsBK0ON5+TIZliLVt1V5RXZ6pTj9B0ElE7f4AGFmMxZxfyFMcLm2uvE9PP8FcAYQoZyu//U7g3N4dT9vgAAAA==[0m[Pipeline] // stage
[8mha:////4AJAGqZwaYqyNNZtbc8g6ZE112gHcOMJEUqcQ3uLTkOfAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJBqEqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A361JXbBAAAA[0m[Pipeline] stage
[8mha:////4PP68w9I/uoPNBdRzp82S+tTLGAqcpDjYGyOG/BwZs8HAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycogU2BDqxBp14QShCSFp9H9JUtKJE3E17kCgEhMeLNuS9Z4vqIKHhrxmVmFvMHSGDW7UObFEvr84SszSmXWEgZxirUotSXWca0tRwayihAWHpcLOUTCoI6y4FXdRO4G6PkWftwOH0sgbPKDgmRyFj98yeVhfRxkIf5y/AJiGCOV2n223+TyrN7xWSV3CAAAA[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4BsNiMtY/8UtmXAIxp6NV4VtqKiaHUkJeJGRvFkfJcU+AAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICtIgqreWGF5jYGCfWnbEdkooX8TX+QCASFVusdraZ5wuWKcKRo8XWUOcoNQ6D7+20cODYXTwP2PIZG6bE3qA0g2Rt6hklZwNzihIWAlaGGs/Jkc2wFq26q8orstUpx+k7CCidvsEDCjGZs4r5C2OEzbXXienn+SuAMWQo99tP7cIbaOP4NL4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.7.19"},"dpdk_dir":"/opt/APP/utility/DPDK/dpdk-21.11/","vm_image_conf":{"image1":"ubuntu20.04.img","image2":"ubuntu20.05.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.115.71","pf1":"enp137s0f0","os":"UBUNTU"},"pkg_generator":{"host":"10.239.115.16","pf1":"ens801f0"}}'
[8mha:////4AyFPX+UZcZf/6aTNXdj+lWE0Z2b4chbTZ9e7eCHFoeRAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKC0iIqWisNLzCJMXasO8d2cCpexNf4A4FIVGyx2tlmni9YxwBHDhqtot5QbA16N+p5YebQXx1ntHzBlimyU9io3HCnTgs2nBQsKUpYCdgoah1HQzrBVlh5l5WTpKtzCvN3EFCaboAHFGI2JxnSF6YAu9vYRaaf568AJu8TlHX9qf0b7E/nQb4AAAA=[0m[Pipeline] }
[8mha:////4AFg/aWh/jHSmrlH6sy1kKjz0TFBfVRN+miPQW2LyX/uAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIioEIoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcrdPtf28AEud0rYvgAAAA==[0m[Pipeline] // stage
[8mha:////4Bhd+KV1EsVM3bnXBHnMClLv0EnBj/3vPQbr+7HWEbmAAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjoXSIitZywwtMYowT6y7YF5KKF/E1/oAhEhVb7ay0mucLyhThQNHJzmLvMTVeDmF0ucmJYn8JNMmOzrIhTBSs1HbS1NrjgprYwhJRwErB2mITKHl0DBvVmbupgkFXnTjmba+g8O0NHiBUNrOJ/IU5wvY6tonw5/krgHlgEDVDsas/x/INPbxT+8EAAAA=[0m[Pipeline] stage
[8mha:////4P13TIuP26wSoF8WyLGiU1qV0Q6Z+QylTska1uSfBb7mAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUQjAhpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD/hYjrHCAAAA[0m[Pipeline] { (run nic case)
[8mha:////4FM3oozjxgRBqafqHchkdSyZ1a7wU/MTBVKRbtZrSwmtAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiIUGDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAEIWoEpvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 17 items / 16 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-03-12 02:53:55 - NicTestFixture.py - [line:28] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[ubuntu20.05.img-ubuntu20.04.img] case ============================
2022-03-12 02:53:55 - test_network_sriov_stress.py - [line:28] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-03-12 02:53:55 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-12 02:53:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-12 02:53:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-12 02:53:56 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-12 02:53:56 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-12 02:53:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-12 02:53:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-12 02:53:59 - sshInstance.py - [line:46] - root - INFO - modinfo ice | grep ^version:
2022-03-12 02:53:59 - sshInstance.py - [line:50] - root - INFO - version:        1.7.19

2022-03-12 02:54:07 - sshInstance.py - [line:46] - root - INFO - rmmod ice
2022-03-12 02:54:10 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:54:18 - sshInstance.py - [line:46] - root - INFO - modprobe ice
2022-03-12 02:54:21 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:54:29 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0 up
2022-03-12 02:54:29 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:54:37 - sshInstance.py - [line:46] - root - INFO - modinfo ice
2022-03-12 02:54:37 - sshInstance.py - [line:50] - root - INFO - filename:       /lib/modules/5.4.0-26-generic/updates/drivers/net/ethernet/intel/ice/ice.ko
firmware:       updates/intel/ice/ddp/ice.pkg
version:        1.7.19
license:        GPL v2
description:    Intel(R) Ethernet Connection E800 Series Linux Driver
author:         Intel Corporation, <linux.nics@intel.com>
srcversion:     24B39F67002570BEF7063AE
alias:          pci:v00008086d00001888sv*sd*bc*sc*i*
alias:          pci:v00008086d0000151Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Fsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000189Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001899sv*sd*bc*sc*i*
alias:          pci:v00008086d00001898sv*sd*bc*sc*i*
alias:          pci:v00008086d00001897sv*sd*bc*sc*i*
alias:          pci:v00008086d00001894sv*sd*bc*sc*i*
alias:          pci:v00008086d00001893sv*sd*bc*sc*i*
alias:          pci:v00008086d00001892sv*sd*bc*sc*i*
alias:          pci:v00008086d00001891sv*sd*bc*sc*i*
alias:          pci:v00008086d00001890sv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Asv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001599sv*sd*bc*sc*i*
alias:          pci:v00008086d00001593sv*sd*bc*sc*i*
alias:          pci:v00008086d00001592sv*sd*bc*sc*i*
alias:          pci:v00008086d00001591sv*sd*bc*sc*i*
depends:        auxiliary
retpoline:      Y
name:           ice
vermagic:       5.4.0-26-generic SMP mod_unload 
parm:           debug:netif level (0=none,...,16=all) (int)
parm:           fwlog_level:FW event level to log. All levels <= to the specified value are enabled. Values: 0=none, 1=error, 2=warning, 3=normal, 4=verbose. Invalid values: >=5
 (ushort)
parm:           fwlog_events:FW events to log (32-bit mask)
 (ulong)

2022-03-12 02:54:45 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0
2022-03-12 02:54:45 - sshInstance.py - [line:50] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 8  bytes 656 (656.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-12 02:54:53 - sshInstance.py - [line:46] - root - INFO - ifconfig ens801f0
2022-03-12 02:54:53 - sshInstance.py - [line:50] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 2439794605  bytes 3691360649938 (3.3 TiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 101733725  bytes 6736473649 (6.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-12 02:54:58 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 1 time when VF is Assigned VM guest.*************************************************************


2022-03-12 02:55:01 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-12 02:55:01 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-12 02:55:06 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-12 02:55:06 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-12 02:55:06 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 89
2022-03-12 02:55:12 - sshInstance.py - [line:94] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
Last login: Sat Mar 12 01:42:34 2022 from 10.239.182.99
root@unassigned-hostname:~# ifconfig enp137s0f0 192.168.10.10
root@unassigned-hostname:~# 
2022-03-12 02:55:15 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-12 02:55:18 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-12 02:55:18 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:23 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-12 02:55:26 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-12 02:55:26 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-12 02:55:31 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-12 02:55:34 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-12 02:55:34 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:42 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-12 02:55:43 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:48 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-12 02:55:51 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-12 02:55:52 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:57 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-12 02:55:57 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-12 02:56:00 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-12 02:56:00 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

F2022-03-12 02:56:08 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-12 02:56:08 - sshInstance.py - [line:50] - root - INFO - bash: virsh: command not found

2022-03-12 02:56:13 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-12 02:56:16 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-12 02:56:16 - sshInstance.py - [line:50] - root - INFO - bash: virsh: command not found

2022-03-12 02:56:21 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest2
2022-03-12 02:56:24 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-12 02:56:24 - sshInstance.py - [line:50] - root - INFO - bash: virsh: command not found

2022-03-12 02:56:29 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-12 02:56:32 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-12 02:56:32 - sshInstance.py - [line:50] - root - INFO - bash: virsh: command not found

2022-03-12 02:56:37 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest1
2022-03-12 02:56:40 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-12 02:56:42 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:56:50 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0 0
2022-03-12 02:56:50 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:56:58 - sshInstance.py - [line:46] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-12 02:56:58 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:57:03 - test_network_sriov_stress.py - [line:81] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-12 02:57:03 - test_network_sriov_stress.py - [line:83] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-12/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[ubuntu20.05.img-ubuntu20.04.img]_02_53_55
2022-03-12 02:57:06 - sshInstance.py - [line:46] - root - INFO - rm -f /tmp/dmesg.log
2022-03-12 02:57:06 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:57:11 - test_network_sriov_stress.py - [line:85] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-12 02:57:11 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2022-03-12 02:57:11 - NicTestFixture.py - [line:32] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[ubuntu20.05.img-ubuntu20.04.img] case ============================
                  [100%]

=================================== FAILURES ===================================
_ Test_Network_SRIOV_Stress.test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[ubuntu20.05.img-ubuntu20.04.img] _

self = <Tests.testNics.test_network_sriov_stress.Test_Network_SRIOV_Stress object at 0x7f2b61177cc0>
vm1_image = 'ubuntu20.05.img', vm2_image = 'ubuntu20.04.img'

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSriovStress
    @pytest.mark.nicSimple
    @pytest.mark.nicSriov5times
    @pytest.mark.parametrize('vm1_image, vm2_image', vm_image)
    def test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest(self, vm1_image, vm2_image):
        for j in range(1, 6):
            logging.info(
                "\n\n********************************************************Stress Disable and Enable for {} time when VF is Assigned VM guest.*************************************************************\n\n".format(
                    j))
            # get pf id: 0000:xx:xx.x
            pf_id, status = self.get_pf_id(self.pf_name)
            # transfer pf_id to 0000\:00\:00.0 format
            if not pf_id:
                assert 200 == status
            else:
                pf_id_format = pf_id.replace(":", "\\:", 2)
                logging.info("format_pf_id is {}".format(pf_id_format))
                # get the number key of pf_id
                pf_id_key = pf_id.split(":")[1]
                logging.info("pf_id_key: {}".format(pf_id_key))
                # assign IP to PF in SUT and sut2
                pf_ip = "192.168.10.10"
                sut2_ip = "192.168.10.11"
                assert 200 == self.assign_ip_ehternet(self.session, pf_ip, self.pf_name)
                # virtulize the pf to vfs
                assert 200 == self.virtualize_pf_to_vfs(self.pf_name)
                # get vf ids
                vf_id_list, status = self.get_vf_ids(pf_id_key)
                vf_id_list = [i.strip() for i in vf_id_list]
                if status == 200:
                    # modprobe vfio-pci
                    assert 200 == self.modprobe("vfio-pci")
                    # bind vf with vfio-pci
                    host_dev_id_list, status = self.bind_vf_to_vfio_pci(vf_id_list)
                    assert 200 == status
                else:
                    assert 200 == status
                # get vF device ida
                # tmp code
                cmd = "lspci -Dnn | grep %s" % (vf_id_list[1])
                logging.info(cmd)
                vf_device_id = ""
                vf_device_id_rs, error = self.ssh_sut.ssh_exec_cmd(self.client, cmd)
                if vf_device_id_rs:
                    st = False
                    vf_device_id = None
                    for one in vf_device_id_rs.split(" "):
                        one = " " + one + " "
                        if one.find(" [") > -1 and one.find("] ") > -1 and one.find(":") > -1:
                            vf_device_id = one.replace("[", "").replace("]", "").strip()
>                   if not vf_device_id: assert 200 == 400, "vf_device_id is not found"
E                   AssertionError: vf_device_id is not found
E                   assert 200 == 400

npxtester/Tests/testNics/test_network_sriov_stress.py:431: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-03-12 02:53:55 - NicTestFixture.py - [line:28] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[ubuntu20.05.img-ubuntu20.04.img] case ============================
2022-03-12 02:53:55 - test_network_sriov_stress.py - [line:28] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-03-12 02:53:55 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-12 02:53:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-12 02:53:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-12 02:53:56 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-12 02:53:56 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-12 02:53:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-12 02:53:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-12 02:53:59 - sshInstance.py - [line:46] - root - INFO - modinfo ice | grep ^version:
2022-03-12 02:53:59 - sshInstance.py - [line:50] - root - INFO - version:        1.7.19

2022-03-12 02:54:07 - sshInstance.py - [line:46] - root - INFO - rmmod ice
2022-03-12 02:54:10 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:54:18 - sshInstance.py - [line:46] - root - INFO - modprobe ice
2022-03-12 02:54:21 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:54:29 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0 up
2022-03-12 02:54:29 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:54:37 - sshInstance.py - [line:46] - root - INFO - modinfo ice
2022-03-12 02:54:37 - sshInstance.py - [line:50] - root - INFO - filename:       /lib/modules/5.4.0-26-generic/updates/drivers/net/ethernet/intel/ice/ice.ko
firmware:       updates/intel/ice/ddp/ice.pkg
version:        1.7.19
license:        GPL v2
description:    Intel(R) Ethernet Connection E800 Series Linux Driver
author:         Intel Corporation, <linux.nics@intel.com>
srcversion:     24B39F67002570BEF7063AE
alias:          pci:v00008086d00001888sv*sd*bc*sc*i*
alias:          pci:v00008086d0000151Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Fsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000189Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001899sv*sd*bc*sc*i*
alias:          pci:v00008086d00001898sv*sd*bc*sc*i*
alias:          pci:v00008086d00001897sv*sd*bc*sc*i*
alias:          pci:v00008086d00001894sv*sd*bc*sc*i*
alias:          pci:v00008086d00001893sv*sd*bc*sc*i*
alias:          pci:v00008086d00001892sv*sd*bc*sc*i*
alias:          pci:v00008086d00001891sv*sd*bc*sc*i*
alias:          pci:v00008086d00001890sv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Asv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001599sv*sd*bc*sc*i*
alias:          pci:v00008086d00001593sv*sd*bc*sc*i*
alias:          pci:v00008086d00001592sv*sd*bc*sc*i*
alias:          pci:v00008086d00001591sv*sd*bc*sc*i*
depends:        auxiliary
retpoline:      Y
name:           ice
vermagic:       5.4.0-26-generic SMP mod_unload 
parm:           debug:netif level (0=none,...,16=all) (int)
parm:           fwlog_level:FW event level to log. All levels <= to the specified value are enabled. Values: 0=none, 1=error, 2=warning, 3=normal, 4=verbose. Invalid values: >=5
 (ushort)
parm:           fwlog_events:FW events to log (32-bit mask)
 (ulong)

2022-03-12 02:54:45 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0
2022-03-12 02:54:45 - sshInstance.py - [line:50] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 8  bytes 656 (656.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-12 02:54:53 - sshInstance.py - [line:46] - root - INFO - ifconfig ens801f0
2022-03-12 02:54:53 - sshInstance.py - [line:50] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 2439794605  bytes 3691360649938 (3.3 TiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 101733725  bytes 6736473649 (6.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


----------------------------- Captured stdout call -----------------------------
2022-03-12 02:54:58 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 1 time when VF is Assigned VM guest.*************************************************************


2022-03-12 02:55:01 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-12 02:55:01 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-12 02:55:06 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-12 02:55:06 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-12 02:55:06 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 89
2022-03-12 02:55:12 - sshInstance.py - [line:94] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
Last login: Sat Mar 12 01:42:34 2022 from 10.239.182.99
root@unassigned-hostname:~# ifconfig enp137s0f0 192.168.10.10
root@unassigned-hostname:~# 
2022-03-12 02:55:15 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-12 02:55:18 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-12 02:55:18 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:23 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-12 02:55:26 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-12 02:55:26 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-12 02:55:31 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-12 02:55:34 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-12 02:55:34 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:42 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-12 02:55:43 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:48 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-12 02:55:51 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-12 02:55:52 - sshInstance.py - [line:50] - root - INFO - 
2022-03-12 02:55:57 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-12 02:55:57 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-12 02:56:00 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-12 02:56:00 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

------------------------------ Captured log call -------------------------------
INFO     root:test_network_sriov_stress.py:389 

********************************************************Stress Disable and Enable for 1 time when VF is Assigned VM guest.*************************************************************


INFO     root:sshInstance.py:46 /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
INFO     root:sshInstance.py:50 0000:89:00.0

INFO     root:test_network_sriov_stress.py:110 pf_id was found: 0000:89:00.0

INFO     root:test_network_sriov_stress.py:397 format_pf_id is 0000\:89\:00.0

INFO     root:test_network_sriov_stress.py:400 pf_id_key: 89
INFO     root:sshInstance.py:94 Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
Last login: Sat Mar 12 01:42:34 2022 from 10.239.182.99
root@unassigned-hostname:~# ifconfig enp137s0f0 192.168.10.10
root@unassigned-hostname:~# 
INFO     root:test_network_sriov_stress.py:301 Successfully assigned ip to ethernet: enp137s0f0
INFO     root:sshInstance.py:46 echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
INFO     root:sshInstance.py:50 
INFO     root:test_network_sriov_stress.py:124 Successfully Virtualized PF to VFs.
INFO     root:sshInstance.py:46  lspci | grep "Virtual" | awk '{print $1;}'list
INFO     root:sshInstance.py:50 89:01.0
89:01.1
89:01.2
89:01.3

INFO     root:test_network_sriov_stress.py:138 Succssfully Gotten vf_id list:

INFO     root:sshInstance.py:46 modprobe vfio-pci

INFO     root:sshInstance.py:50 
INFO     root:sshInstance.py:46 /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
INFO     root:sshInstance.py:50 
INFO     root:test_network_sriov_stress.py:171 Successfuly bound VF to vfio-pci
INFO     root:sshInstance.py:46 /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
INFO     root:sshInstance.py:50 
INFO     root:test_network_sriov_stress.py:171 Successfuly bound VF to vfio-pci
INFO     root:test_network_sriov_stress.py:421 lspci -Dnn | grep 89:01.1
INFO     root:sshInstance.py:46 lspci -Dnn | grep 89:01.1
INFO     root:sshInstance.py:50 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:69
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:107
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:107: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:382
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:382: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:383
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:383: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:500
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:500: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:799
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:799: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:85
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:85: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:110
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:134
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:134: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:166
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:166: PytestUnknownMarkWarning: Unknown pytest.mark.driveInstall - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.driveInstall

npxtester/Tests/testNics/test_nics.py:206
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:206: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:232
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:232: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_network_sriov_stress.py::Test_Network_SRIOV_Stress::test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[ubuntu20.05.img-ubuntu20.04.img]
========== 1 failed, 16 deselected, 16 warnings in 196.08s (0:03:16) ===========
[8mha:////4JadWPtzQEFx9CzixTlssp2D6uuorSv3wnTa6h1wuGrmAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiISEKRJXWcsMLTGyME+vO2A5JxYv4Gn8gEImKLVY728zzBcsU4cjRYmuoc5Qah8H3dlo4cOwungds+YwNU2JvUJpBsjb1jJKzgTlFCQsBK0ON5+TIZliLVt1V5RXZ6pTj9B0ElE7f4AGFmMxZxfyFMcLm2uvE9PP8FcAYQoZyu//U7g3QKkBhvgAAAA==[0m[Pipeline] }
[8mha:////4Goa9rseL6gCK8+j51lIjOVJAeE64BF71YmRtb4KtKxFAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiIQokRJXWcsMLTGyME+vO2A5JxYv4Gn8gEImKLVY728zzBcsU4cjRYmuoc5Qah8H3dlo4cOwungds+YwNU2JvUJpBsjb1jJKzgTlFCQsBK0ON5+TIZliLVt1V5RXZ6pTj9B0ElE7f4AGFmMxZxfyFMcLm2uvE9PP8FcAYQoZyu//U7g2TqBHnvgAAAA==[0m[Pipeline] // stage
[8mha:////4AK444efjkykafC1HGbgxn6nPSQkL/HNauaXETS7R91oAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIRqEqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A+OvzUfBAAAA[0m[Pipeline] stage
[8mha:////4EqpMgzslAWPDRl1bp4feAKRfPoiBNazp8uDBURuqJGsAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycogUwYRQJ9aoCycITQhJo/9LkpJOnIircQcClZjwYNmWrPd8QRU8NOQ1swp7g6EzbHCjzokl8v3FUWKWzqwjDOQUa1VqSarjXFuKCmYVJSw4LBV2joJBHWHFrbiL2gnU9Sn6vB04lEbe4AEFz+QofPyWycP6OspA+OP8BcA0RCi3+2y7zedZvQH4j6CMwgAAAA==[0m[Pipeline] { (collect logs)
Stage "collect logs" skipped due to earlier failure(s)
[8mha:////4M0P44+7KKMsBaEAgOPFTC6kJ9gOLfRk2xH1/27GZQUTAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICokRUaS03vMDExjix7oztkFS8iK/xBwKRqNhitbPNPF+wTBGOHC22hjpHqXEYfG+nhQPH7uJ5wJbP2DAl9galGSRrU88oORuYU5SwELAy1HhOjmyGtWjVXVVeka1OOU7fQUDp9A0eUIjJnFXMXxgjbK69Tkw/z18BjCFkKPe7T23ftwcfeL4AAAA=[0m[Pipeline] }
[8mha:////4ME5TlbHmzr5qdpFH/mkGWKP7XmPx2rIKtfLHFMqug00AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIihQohqrRWGl5gYmOcWHfGdkgqXsTX+AMWkajYYrWzzbzesI4BThwM9poGS7Gz6N1o8sKJw3B1PGHPF+yYIjuNrZ5aVrpZsOWkYUlRwkrARlPnOFoyCbailw9ZOUmmOqeQv6OA0qo7PKEQ2ZxkSF+YA+xuo4pMP89fAczeJyj3da768AH8/EyevgAAAA==[0m[Pipeline] // stage
[8mha:////4Fe3eqQpZrcpnjrK1jN1d/MbMZ7ufuA8IOyekGcoBhh7AAAAoh+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIU2DlWthkaTxAhxkBmF5NFqDyRV/MOMjJj5a/+e817vbFOEUeOTrWWOk+p9qoPg5ufGjl218CjavmiaqbEwarKjhU39rRgxWKxLMux0thYqgMnT06w1a15mCIYcsVZ4uwOGrlv7ngi03NZTJQvTBG729Akpl/nbwBT3wvyshRk+w9e8C3YvQAAAA==[0m[Pipeline] }
[8mha:////4Fl8XDDe7BXdXsXj7uDbwTY/cDAcq+nOEdlmagh5XH3DAAAAoh+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIU2DlWthkaTxAhxkBmF5NFqDyRV/MOMjJj5a/+e817vbFOEUeOTrWWOk+p9qoPg5ufGjl218CjavmiaqbEwarKjhU39rRgxWKxLMux0thYqgMnT06w1a15mCIYcsVZ4uwOGrlv7ngi03NZTJQvTBG729Akpl/nbwBT3wvyci/Iyg/tuWn0vQAAAA==[0m[Pipeline] // node
[8mha:////4Oc/DLR66YI1esGEMsMY02BciEZxcpD10WBUR6yLP96pAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKRQFSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLcXZNsPy6AOvr0AAAA=[0m[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
